src/FakeAuth/AuthExtension.cs

```diff
@@ -20,10 +25,10 @@ public static void UseFakeAuth<TProfile>(this IServiceCollection services) where
 
 		public static void UseFakeAuth(this IServiceCollection services, Action<FakeAuthOptions> options)
 		{
-			services.AddAuthentication(FakeAuthConst.SchemaName)
-			.AddScheme<FakeAuthOptions, FakeAuthHandler>(FakeAuthConst.SchemaName, null);
+			services.AddAuthentication(FakeAuthDefaults.SchemaName)
```

Comment:
This is problematic if someone has already called `AddAuthentication(...)` with different args. IMHO the `UseFakeAuth(...)` method needs to go the way of the dodo.

Reply:
Ok, did a little bit of research here, my original concern about multiple calls to `.AddAuthentication(...)` is overblown `</chicken-little>`. It's not uncommon for there to be multiple calls to `.AddAuthentication()`. Calling it with no arguments is pretty benign, so if we had a method on `IServiceCollection` called `.AddOrUseOrEngageFakeAuth()` (pending the naming convention decision 😇 ) that internally set the default scheme, that seems reasonable. We just need to make sure that it's clearly documented THIS WILL SET THE DEFAULT SCHEME. As long as we also have `.AddFakeAuth(...)` on `AuthenticationBuilder` so that FakeAuth can live amongst other auth schemes, I think that covers both the one line of code scenario as well as FakeAuth + other auth providers scenario with the appropriate level of control.